src/resources/views/base/inc/sidebar.blade.php

```diff
@@ -25,10 +25,19 @@
   <script type="text/javascript">
     // Save default sidebar class
     let sidebarClass = (document.body.className.match(/sidebar-(sm|md|lg|xl)-show/) || ['sidebar-lg-show'])[0];
+    let sidebarTransition = value => document.querySelector('.sidebar').style.transition = value || '';
```

Comment:
I see the query selector here is `.sidebar` - any chance people are using `.sidebar` somewhere else on page too, and we'd be eliminating their transition there?

I don't know, I'm just asking to be safe.

Reply:
I did that "broad" query because on CSS, the transition is applied to that class with no other rules;
```css
.app-footer, .app-header, .aside-menu, .main, .sidebar {
    transition: margin-left .25s,margin-right .25s,width .25s,flex .25s;
}
```

But I agree, people may be using a sidebar on their dashboard for instance, we never know 🤷‍♂️ 

I'm going to change to `.app-body > .sidebar` since it's safer.